runtime/flangrti/ktrap.c

```diff
@@ -50,7 +50,11 @@ __ktrap(void)
         excepts |= FE_UNDERFLOW;
       if (bv & 0x100)
         excepts |= FE_INEXACT;
+#ifdef TARGET_OSX
```

Comment:
Tried this on my mac. Neither of `__fenv_feenableexcept()` or `feenableexcept()` could be found...

Reply:
Looks like the function was moved from flangrti to libpgmath. https://github.com/flang-compiler/flang/blob/master/runtime/libpgmath/lib/common/fltfenv.c